apps/nextjs-website/src/components/organisms/ChatbotHistoryDetailLayout/ChatbotHistoryDetailLayout.tsx

```diff
@@ -61,6 +69,7 @@ const ChatbotHistoryDetailLayout = ({
           startIcon={<Delete />}
           color='error'
           sx={{ display: { xs: 'none', xl: 'flex' } }}
+          onClick={() => onDeleteChatSession(queries[0].sessionId)}
```

Comment:
The delete button disappear when the viewport width is smaller than 1590px:
[screencast-localhost_3000-2024_10_07-15_08_05.webm](https://github.com/user-attachments/assets/0e999349-69be-460a-a655-98886ae8e9d3)